src/latexeditorview.cpp

```diff
@@ -370,7 +372,8 @@ bool DefaultInputBinding::contextMenuEvent(QContextMenuEvent *event, QEditor *ed
 			bool removePreviewActionFound = false;
 			foreach (QAction *act, baseActions) {
 				if (act->objectName().endsWith("removePreviewLatex")) {
-					act->setData(posInDocCoordinates);
+					isPicMenu = true;
+					act->setData( QVariantList() << QVariant(posInDocCoordinates) << QVariant(isPicMenu) );
```

Comment:
there should not be a necessity to this list.
Especially as the first element type basically depends on the second makes the code hard to read.
Simplest solution is, to use already line/col at this point instead of coordinate which makes a later distinction unnecessary.